content/en/docs/tasks/manage-kubernetes-objects/declarative-config.md

```diff
@@ -950,20 +950,20 @@ configuration involves several manual steps:
      kubectl get <kind>/<name> -o yaml > <kind>_<name>.yaml
      ```
 
-1. Manually remove the `status` field from the configuration file.
+2. Manually remove the `status` field from the configuration file.
```

Comment:
pls avoid manually number the bullets.
Explicit numbering will make future insertion or deletion awkward.

Reply:
Just in time, I'd like to ask this question.
Thanks a lot 😁